compiler-core/src/javascript/tests/bit_arrays.rs

```diff
@@ -50,6 +50,24 @@ fn float() {
         r#"
 fn go() {
   <<1.1:float>>
+  <<1.1:float-big>>
+  <<1.1:float-little>>
```

Comment:
Each test should only test one thing, so make a new test for each of these new examples please :pray:

Reply:
Done!